tests/scripts/create-dev-cluster.sh

```diff
@@ -54,21 +58,28 @@ show_info() {
     fi
     echo "==========================="
     echo " "
-    echo " *** To start using your rook cluster please set the following alias: "
+    echo " *** To start using your rook cluster please set the following env: "
+    echo " "
+    echo "   > eval \$($MINIKUBE docker-env)"
+    echo "   > alias kubectl=\"$KUBECTL"\"
+    echo " "
+    echo " *** To access the new cluster with k9s: "
```

Comment:
k8s or kubernetes instead of k9s.

Reply:
k9s here referes to the tool [k9s](https://github.com/derailed/k9s)